givitsite/coordinate/views.py

```diff
@@ -1,3 +1,131 @@
-from django.shortcuts import render
+import sys
+import os
+from django.shortcuts import render, redirect
+from django.http import HttpRequest, HttpResponse, Http404
+from django.contrib.auth.models import User
+from friendreq.models import ItemsFound
+from friendreq.models import ItemRequest
+from django.conf import settings
+from . models import CoordinatedItems
+
 
 # Create your views here.
+def coordinator_create_view(request):
+    
+    if request.method == 'GET':
+        itemsToCoordinate = []
+        matchedItems = ItemsFound.objects.filter(match=True)
+        openRequests = ItemRequest.objects.filter(status='open')
+        users = User.objects.all()
+        
+
+        # mock data for testing:
```

Comment:
You can't push a commented code
Please remove those lines and a relevant comment with the issue for that addition

Reply:
Removed the comments all together - they are no related to any issue